docs/user/how-to/run-container.rst

```diff
@@ -13,3 +13,5 @@ To pull the container from github container registry and run::
     $ docker run ghcr.io/DiamondLightSource/blueapi:main --version
 
 To get a released version, use a numbered release instead of ``main``.
+Note: running with Podman you might get `Error: repository name must be lowercase.`. Re-running the command as `podman run ghcr.io/diamondlightsource/blueapi:main --version` should solve this.
+
```

Comment:
```suggestion
    $ docker run ghcr.io/diamondlightsource/blueapi:main --version

To get a released version, use a numbered release instead of ``main``.
```

Reply:
Assuming that docker is also OK with the lowercase, why not just make it the recommended form?

Reply:
yeah of course!

Reply:
yeah of course